packages/driver/cypress/support/utils.ts

```diff
@@ -21,42 +22,30 @@ export const findReactInstance = function (dom) {
 }
 
 export const clickCommandLog = (sel, type) => {
-  return cy.wait(10)
+  // trigger the LONG_RUNNING_THRESHOLD to display the command line
+  // this adds time to test but makes a more accurate test as React 18+ does not rerender when setting internals
+  return cy.wait(2000)
   .then(() => {
-    return withMutableReporterState(() => {
-      const commandLogEl = getCommandLogWithText(sel, type)
-      const reactCommandInstance = findReactInstance(commandLogEl[0])
+    const commandLogEl = getCommandLogWithText(sel, type)
+
+    const reactCommandInstance = findReactInstance(commandLogEl[0])
 
-      if (!reactCommandInstance) {
-        assert(false, 'failed to get command log React instance')
-      }
+    if (!reactCommandInstance) {
+      assert(false, 'failed to get command log React instance')
+    }
 
-      reactCommandInstance.props.appState.isRunning = false
-      const inner = $(commandLogEl).find('.command-wrapper-text')
+    reactCommandInstance.props.appState.isRunning = false
+    const inner = $(commandLogEl).find('.command-wrapper-text')
 
-      inner.get(0).click()
+    inner.get(0).click()
 
+    cy.wait(10).then(() => {
       // make sure command was pinned, otherwise throw a better error message
       expect(cy.$$('.runnable-active .command-pin', top?.document).length, 'command should be pinned').ok
     })
   })
 }
 
-export const withMutableReporterState = (fn) => {
-  // @ts-ignore
-  top?.UnifiedRunner.MobX.configure({ enforceActions: 'never' })
-
-  const currentTestLog = findReactInstance(cy.$$('.runnable-active', top?.document)[0])
-
-  currentTestLog.props.model._isOpen = true
```

Comment:
this no longer works (hence why we are waiting the `LONG_RUNNING_THRESHOLD` above).

The model does get updated in `<Attempt/>`, but the prop passed into `<Collapsible/> in `componentDidUpdate` still says `false`, which seems like a bug.

Normally I would file an issue for this, but I would argue that waiting the full 2 seconds for the attempt to open is a better test since this is what the user would exactly experience and better tests our code, even though it adds a few seconds to each test leveraging this. We don't do this is many tests (maybe 7 tests total) so it seems fine.